src/main/java/com/rultor/agents/github/qtn/QnAskedBy.java

```diff
@@ -127,6 +117,34 @@ public Object apply(final String input) {
     }
 
     /**
+     * Format list of commanders with {@code @} prefix, comma-delimited.
+     * @param logins Commanders
+     * @return Comma-delimited names
+     */
+    private String commandersAsDelimitedList(final Collection<String> logins) {
+        return StringUtils.join(
+            Iterables.transform(
+                Iterables.filter(
+                    logins,
+                    new Predicate<Object>() {
+                        @Override
+                        public boolean apply(final Object input) {
+                            return !"rultor".equals(input);
```

Comment:
I would rather use `comment.issue().repo().github().users().self()` instead of hardcoded `"rultor"` string. With the hardcoded string we assume that our login to Github is always the same. This may be so for now, but the code doesn't look self-explainable, since the reader may not understand what this magic "rultor" string means.